### ISSUE TITLE
[Feature] Allow tooltips text to be customised

### ISSUE THREAD

**Issue by accforgithubtest** [lovasoa/SQLpage#217]
With all the charts in sqlpage, the tooltips seem to be only auto-generated.

It would be nice if the tooltips can be customized, allowing users to add extra data / context to the tooltip, (using data from columns or other static text) that are not necessarily required for the chart, but is extra info that can be passed on to a user via tooltips.

**Comment by DSMejantel** [comment]
Hi, it's possible to add some informations like this : 
```
select 
    SUBSTR(aesh.aesh_firstname, 1, 1) ||'. '|| aesh_name as label,
    SUBSTR(eleve.prenom, 1, 1) ||'. '||eleve.nom||' ('||eleve.classe||')' as series,
    sum(suivi.temps/mut)        as value
        FROM suivi JOIN...
```
![Capture-20240207072712-379x276](https://github.com/lovasoa/SQLpage/assets/149260856/3a1dbabd-4acb-4599-b500-cdade0d91935)

**Comment by accforgithubtest** [comment]
Hi @DSMejantel , thanks for your response. 

In this example is the x-axis the same as the label ? 
According to the docs, looks like label is just an alias for x.

So when I tried to add a ` .... as label` to a chart that already had a timeseries for x, it broke the chart.

Pls let me know if I am missing something.

**Comment by DSMejantel** [comment]
You're right. If we have 'something as x', we can't ask for 'anotherthing as label'.
What I propose with my example, you can concate differents rows in the label with || in relation with the context of the data
like here with firstname and name : 
SUBSTR(aesh.aesh_firstname, 1, 1) ||'. '|| aesh_name as label

**Comment by accforgithubtest** [comment]
Thanks for the reply @DSMejantel. I am on the same page with you about this now.

The feature request is to allow for values that are not the same as the x or y to be passed to the tooltip.
For example x is a time series, y is a dollar amount. Tooltip can provide other extra info (other than the dollar amount / predefined x-axis).
